src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -63,6 +80,25 @@ public virtual Task<BaseConfiguration> GetBaseConfigurationAsync(CancellationTok
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Gets all valid last known good configurations from the cache.
+        /// </summary>
+        /// <returns>A collection of all valid last known good configurations.</returns>
+        internal ICollection<BaseConfiguration> GetValidLkgConfiguraitonFromCache()
```

Comment:
Suggest dropping the postfix: 'Cache'

Reply:
Makes sense, a bit superfluous I suppose, LKG will always be a cache of some sort.